examples/eth1_del_all_routes.yml

```diff
@@ -0,0 +1,16 @@
+---
+interfaces:
```

Comment:
Do we need this section?
If it is dropped, we can show that routes may be removed without specifying the interface itself.

Reply:
We need it as we don't support editing routes without interface yet.

https://nmstate.atlassian.net/browse/NMSTATE-212

Reply:
Ohh, right. Is specifying the name enough?

Reply:
Changed to include name only.